Bastion/src/isaac/bastion/listeners/BastionListener.java

```diff
@@ -1,82 +0,0 @@
-package isaac.bastion.listeners;
-
-import isaac.bastion.Bastion;
-import isaac.bastion.commands.PlayersStates;
-import isaac.bastion.commands.PlayersStates.Mode;
-import isaac.bastion.manager.BastionBlockManager;
-import isaac.bastion.manager.ConfigManager;
-
-import org.bukkit.ChatColor;
-import org.bukkit.event.EventHandler;
-import org.bukkit.event.EventPriority;
-import org.bukkit.event.Listener;
-import org.bukkit.event.block.BlockBreakEvent;
-import org.bukkit.event.block.BlockDispenseEvent;
-import org.bukkit.event.block.BlockFromToEvent;
-import org.bukkit.event.block.BlockPistonExtendEvent;
-import org.bukkit.event.block.BlockPlaceEvent;
-import org.bukkit.event.player.PlayerBucketEmptyEvent;
-import org.bukkit.event.world.StructureGrowEvent;
-
-import vg.civcraft.mc.citadel.events.ReinforcementCreationEvent;
-import vg.civcraft.mc.citadel.reinforcement.PlayerReinforcement;
-
-public final class BastionListener implements Listener {
```

Comment:
It was kinda weird how where all the events were so I renamed this to BastionDamageListener and made it only the events listening for block places etc